action/protocol/staking/candidate.go

```diff
@@ -26,6 +26,7 @@ type (
 		Owner              address.Address
 		Operator           address.Address
 		Reward             address.Address
+		Voter              address.Address
```

Comment:
what's voter? how does it work? what if the ownership be transferred for more than one time?